packages/react-router-native/index.tsx

```diff
@@ -218,14 +232,14 @@ export function useDeepLinking() {
   }, [navigate]);
 }
 
-function trimScheme(url) {
+function trimScheme(url: string) {
   return url.replace(/^.*?:\/\//, '');
 }
 
 /**
  * Prompts the user with an Alert before they leave the current screen.
  */
-export function usePrompt({ message, when = true }) {
+export function usePrompt(message: string, when: boolean = true) {
```

Comment:
`when: boolean = true` - typing this should not be necessary. 
TS infer the type from default value for simple types.

Reply:
Great suggestion, thanks. I'll use that.